book/website/project-design/community-growth-sustainability.md

```diff
@@ -0,0 +1,239 @@
+# Open Source Community Building Guide
```

Comment:
1. I would change the title of this section to something more specific, akin to your title, like "Community Roadmaps"
2. Don't forget to add the specific syntax at the top of your chapter, as outlined in our [Style guide](https://book.the-turing-way.org/community-handbook/style/style-crossref).
```suggestion
(pd-community-roadmap)=
# Open Source Community Building Guide
```

Reply:
I went ahead and updated this in the latest commit, let me know if that looks to be correct and if I'm doing the subchapter syntax correct :)